cardano-node/app/cardano-cli.hs

```diff
@@ -461,6 +469,19 @@ parseFlag def active optname desc =
   flag def active $ long optname <> help desc
 
 
+parseTargetNodeAddress :: String -> String -> Parser NodeAddress
```

Comment:
how can we indicate multiple target hosts? it is not "many" anymore.

Reply:
Yes, it's `some` now (one or more): https://github.com/input-output-hk/cardano-node/pull/330/files#diff-deb1de1e23655df9067b46df271864c6R342-R343